docs/manual/en/introduction/Color-management.html

```diff
@@ -0,0 +1,335 @@
+			color space instead, indicated with <i>texture.encoding = LinearEncoding</i>.
+		</li>
+		<li>
+			<b>Non-color textures:</b> Textures that do not store color information (like .normalMap
+			or .roughnessMap) do not have an associated color space, and generally use the (default) texture
+			annotation of <i>texture.encoding = LinearEncoding</i>. In rare cases, non-color data
+			may be represented with other nonlinear encodings for technical reasons.
+		</li>
+	</ul>
+
+	<blockquote>
+		<p>
+			⚠️ <i><b>WARNING:</b> [page:Scene.fog] and [page:Scene.background] are exceptions to
+			the rule. Both are unaffected by [page:WebGLRenderer.outputEncoding] and so must
+			store RGB components in the renderer's <u>output</u> color space.</i>
+		</p>
+	</blockquote>
+
+	<blockquote>
+		<p>
+			⚠️ <i><b>WARNING:</b> Many formats for 3D models do not correctly or consistently
+			define color space information. While three.js attempts to handle most cases, problems
+			are common with older file formats. For best results, use glTF 2.0 ([page:GLTFLoader])
+			and test 3D models in online viewers early to confirm the asset itself is correct.</i>
+		</p>
+	</blockquote>
+
+	<h3>Working Color Space</h3>
+
+	<p>
+		Rendering (and most other operations) must be done using an open domain linear working
+		color space, in which RGB components are proportional to physical illumination. In
+		three.js, the working color space is Linear-sRGB.
+	</p>
+
+	<h3>Output Color Space</h3>
+
+	<p>
+		Output to a display device, image, or video may involve conversion from the open domain
+		Linear-sRGB working color space to another color space. This conversion may be performed in
+		the main render pass ([page:WebGLRenderer.outputEncoding]), or during post-processing.
+	</p>
+
+	<code>
+renderer.outputEncoding = THREE.sRGBEncoding; // optional with post-processing
+	</code>
+
+	<ul>
+		<li>
+			<b>Display:</b> Colors written to a WebGL canvas for display should be in the sRGB
+			color space.
+		</li>
+		<li>
+			<b>Image:</b> Colors written to an image should use the color space appropriate for
+			the format and usage. Fully-rendered images written to PNG or JPEG textures generally
+			use the sRGB color space. Images containing emission, light maps, or other data not
+			confined to the [0,1] range will generally use the open domain Linear-sRGB color space,
+			and a compatible image format like EXR.
+		</li>
+	</ul>
+
+	<blockquote>
+		<p>
+			⚠️ <i><b>WARNING:</b> Render targets may use either sRGB or Linear-sRGB. sRGB makes
+			better use of limited precision — 8 bits suffice for LDR output to sRGB, whereas ≥12 bits
+			(half float textures) are often required for LDR output to Linear-sRGB. However, the
+			next pipeline stage using the render target may require input in the working color
+			space, Linear-sRGB, incurring modest overhead to perform an additional sRGB → Linear-sRGB
+			conversion.</i>
```

Comment:
TODO: Use of the term "LDR" is ambiguous here, closed domain [0,1] would be clearer.